addons/stock_calendar/tests/test_stock_calendar.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import common
+from datetime import datetime, timedelta
+from odoo import fields
```

Comment:
Please follow guideline : arrange like : 
```
from datetime import datetime, timedelta

from odoo import fields
from odoo.tests import common
``` 
We should try to improve odoo code with remember this type of little bit points. :palm_tree: